### DISCUSSION TITLE
G2 Error.33 on Lathe Mode

### DISCUSSION THREAD

**Discussion by Reprosense** [grblHAL/core#416] [Q&A]
hi guys , i have try to use G2 on my lathe mode , but it shows err.33 instead , can i get a clue what could make it happen ?  here is my video and my gcode
[G2 TEST.txt](https://github.com/grblHAL/core/files/13776849/G2.TEST.txt)

https://github.com/grblHAL/core/assets/135573322/099ca65a-6609-44b1-8a08-8762f9b6e6ca

thanks in advance

**Comment by terjeio** [comment]
Duplicate of [this discussion](https://github.com/grblHAL/STM32F4xx/discussions/153).
